cmake/modules/PytestBenchmark.cmake

```diff
@@ -0,0 +1,21 @@
+include (FindPythonModule)
+
+# Check which Python version ROOT was build against  
+find_program(ROOT_CONFIG_EXECUTABLE root-config)
+exec_program(${ROOT_CONFIG_EXECUTABLE} ARGS "--python-version" OUTPUT_VARIABLE ROOT_PYTHON_VERSION RETURN_VALUE RETVAR)
+set(ROOT_PYTHON_VERSION ${ROOT_PYTHON_VERSION} CACHE INTERNAL "")
+string(REGEX REPLACE "^([0-9]+)\\.[0-9]+.*" "\\1" PYTHON_VERSION_MAJOR "${ROOT_PYTHON_VERSION}")
+
+# FindPython we used to build ROOT against 
+find_package(Python${PYTHON_VERSION_MAJOR} COMPONENTS Interpreter NumPy)
+set(PYTHON_EXECUTABLE "${Python${PYTHON_VERSION_MAJOR}_EXECUTABLE}" CACHE INTERNAL "" FORCE)
+set(PYTHON_INCLUDE_DIRS "${Python${PYTHON_VERSION_MAJOR}_INCLUDE_DIRS}" CACHE INTERNAL "" FORCE)
+set(PYTHON_LIBRARIES "${Python${PYTHON_VERSION_MAJOR}_LIBRARIES}" CACHE INTERNAL "" FORCE)
+set(PYTHON_VERSION_MAJOR "${PYTHON_VERSION_MAJOR}" CACHE INTERNAL "" FORCE)
+set(PYTHON_VERSION_MINOR "${Python${PYTHON_VERSION_MAJOR}_VERSION_MINOR}" CACHE INTERNAL "" FORCE)
+set(NUMPY_FOUND ${Python${PYTHON_VERSION_MAJOR}_NumPy_FOUND} CACHE INTERNAL "" FORCE)
+set(NUMPY_INCLUDE_DIRS "${Python${PYTHON_VERSION_MAJOR}_NumPy_INCLUDE_DIRS}" CACHE INTERNAL "" FORCE)
+
+
+# Find mandatory dependency (for Python benchmarks), but quietly :)
```

Comment:
Does this mean that if Python is found but not pytest (though very unlikely), it would try to run with pytest anyway?

Reply:
pytest will come from requirements.txt, I can't do anything here....
The tests will be not configured without pytest because we have restriction in RB_ADD_PYTESTBENCHMARK function :

`if(PYTEST_FOUND AND (ROOT_pyroot_experimental_FOUND OR ROOT_pyroot_FOUND))....`